src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
```

Comment:
This class should not extend `Figure` if `Figure` is an abstract class used to set behavior. If `Figure` is an interface intended for providing common behavior, then this is correct.